app/routes/_auth.signup.tsx

```diff
@@ -158,12 +158,19 @@ export default function Component() {
               </p>
             )}
 
-            <button type="submit" disabled={isPending}>
+            <button
```

Comment:
Could we make this a reusable component if we can't reuse an existing one? This button seems to be used in both login and signup pages.

Reply:
Thanks! It seems I accidentally removed these attributes! 😆 Will place them back!
Interestingly, I didn't think about reusable buttons explicitly for Auth. However, sounds good to me!